webapp/pages/settings/blocked-users.vue

```diff
@@ -92,12 +96,20 @@ export default {
         avatar: '',
         name: this.$t('settings.blocked-users.columns.name'),
         slug: this.$t('settings.blocked-users.columns.slug'),
+        unblock: this.$t('settings.blocked-users.columns.unblock'),
```

Comment:
Ah, this is exactly the mistake I wanted to avoid with my suggestion in the first comment! :grin: I think there is no `blocked-users.columns.unblock` there is just `blocked-users.columns.unblk` or `blocked-users.unblock`.

Reply:
ok